Update clock offset shutdown language

rmloveland opened the pull request:
Specifically:

- Use less charged language to describe node shutdown during max clock
  offset situations

- Add a link to 'Remove Nodes' to provide more detail about what happens
  during the node shutdown process

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/docs/2618)
<!-- Reviewable:end -->

rmloveland commented:
@jseldess I'm less confident about the link to 'Remove Nodes' but it seemed relevant.  If it's not (or somewhat misleading in this case) I'm happy to remove.

cockroach-teamcity commented:
http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/7c4740ed6bc504d5cf23296b2442c490eea198cf/

bdarnell commented:
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

*[v1.1/architecture/transaction-layer.md, line 77 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2618#-L71dFsJFEj9eXObWXmF-r1-77:-L767wQVTB2IdNUpchMd:bwvx4kw) ([raw file](https://github.com/cockroachdb/docs/blob/7c4740ed6bc504d5cf23296b2442c490eea198cf/v1.1/architecture/transaction-layer.md#L77)):*
<details><summary><i>Previously, jseldess (Jesse Seldess) wrote…</i></summary><blockquote>

I like this change, but I think it might make sense to replace this entire paragraph with the first paragraph in this include: https://github.com/cockroachdb/docs/blob/master/_includes/faq/clock-synchronization_v2.0.html. This include language is newer and, I think, a bit more accurate. 

It's a shame not to use the entire include here as well, but I don't think anything but the first paragraph is necessary.

As for node shutdown, I don't know if it's a "graceful" shutdown when due to clock skew. @bdarnell, can you clarify? 
</blockquote></details>

No, it is deliberately not a graceful shutdown (I'm actually in the process of making it even less graceful). We want to shut everything down as abruptly as possible because anything in progress could be serving inconsistent results. "Crashes" or "halts" would be better than "shuts down" here, and we don't go though any of the process described in the "Remove nodes" page. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2618)*
<!-- Sent from Reviewable.io -->

cockroach-teamcity commented:
http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/93c7388d6fcd5631243a4cdd927831ee2b4d1603/

cockroach-teamcity commented:
http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/499a2b947707c1c1affbec92494d28fb19388ce8/

rmloveland commented:
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

*[v1.1/architecture/transaction-layer.md, line 77 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2618#-L71dFsJFEj9eXObWXmF-r1-77:-L7a3xr25bR_pOObsNJo:b-x64i7v) ([raw file](https://github.com/cockroachdb/docs/blob/7c4740ed6bc504d5cf23296b2442c490eea198cf/v1.1/architecture/transaction-layer.md#L77)):*
<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary><blockquote>

No, it is deliberately not a graceful shutdown (I'm actually in the process of making it even less graceful). We want to shut everything down as abruptly as possible because anything in progress could be serving inconsistent results. "Crashes" or "halts" would be better than "shuts down" here, and we don't go though any of the process described in the "Remove nodes" page. 
</blockquote></details>

@jseldess, I replaced the whole paragraph with the one you suggested in 499a2b94, except that I updated the language as @bdarnell suggested to say it **crashes immediately** (in bold so it's hard to miss :-) )

I also removed the links to 'Remove Nodes' since those are about a different type of shutdown (I don't have the commit SHA handy because I did something weird when fixing conflicts).

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2618)*
<!-- Sent from Reviewable.io -->

bdarnell commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/cockroachdb/docs/2618#-:-L7a50ozCWUOHd2ZXX91:bnfp4nl)*
<!-- Sent from Reviewable.io -->

jseldess commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>, Rich.

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/cockroachdb/docs/2618#-:-L7aDojx-tl6Etd1Dt2T:bxsxuq5)*
<!-- Sent from Reviewable.io -->
